docs/config/index.md

```diff
@@ -4,7 +4,7 @@ title: Configuring Vite
 
 # Configuring Vite
 
-When running `vite` from the command line, Vite will automatically try to resolve a config file named `vite.config.js` inside [project root](/guide/#index-html-and-project-root).
+When running `vite` from the command line, Vite will automatically try to resolve a config file named `vite.config.js` inside [project root](/guide/#index-html-and-project-root).The config file suffix supported by vite is: `.js`, `.mjs`, `.ts`, `.cjs`, `.mts`, `.cts`.
```

Comment:
Maybe we could add a line about TypeScript being supported. As Blu already said, having the list also seems unneeded to me. @bluwy I'm fine with you closing this one too though.
```suggestion
When running `vite` from the command line, Vite will automatically try to resolve a config file named `vite.config.js` inside [project root](/guide/#index-html-and-project-root) (other JS and TS extensions are also supported).
```

Reply:
I think adding a note that `.ts` is supported makes sense